src/render/DrawContext.js

```diff
@@ -419,6 +419,9 @@ define([
 
             // Intentionally not documented.
             this.pixelSizeOffset = 0;
+
+            // Intentionally not documented.
+            this.glExtensionsCache = Object.create(null);
```

Comment:
@strikerM, we were briefly looking at the differences between `Object.create(null)` and `{}`. They are pretty clear with respect to the resulting objects. However, have you made any assessment of the performance gain? More generally, was there something that drove your decision to use `Object.create(null)` here? We would like to understand whether this is a pattern that should be promoted in other places of the framework.


For reference, see the validated response at https://stackoverflow.com/questions/32262809/is-it-bad-practice-to-use-object-createnull-versus

Reply:
As discussed, the performance gain is neglectable in this case. This is however a better pattern for showing that this is a data store (dictionary) and not an object.